spec/unit/database/database.spec.js

```diff
@@ -0,0 +1,20 @@
+var db_config = require("./config");
+describe("Database", function() {
+    describe("Application", function() {
+    	beforeEach(function(){
+			var node_env = process.env.NODE_ENV;
+		});
+
+        it("should be connected to the correct database.", function() {
+        	if (node_env === 'development'){
+            	expect(db_config[node_env].database).toBe('');
+        	} else if (node_env === 'test') {
+        		expect(db_config[node_env].database).toBe('');
```

Comment:
Identifier 'db_config' is not in camel case.<br>Identifier 'node_env' is not in camel case.<br>Mixed double and single quotes.<br>'node_env' is not defined.